azurerm/resource_arm_container_group.go

```diff
@@ -589,6 +620,7 @@ func resourceArmContainerGroupRead(d *schema.ResourceData, meta interface{}) err
 		if address := props.IPAddress; address != nil {
 			d.Set("ip_address_type", address.Type)
 			d.Set("ip_address", address.IP)
+			d.Set("ports", address.Ports)
```

Comment:
Linter was failing here because I needed to flatten the ports and provide this as a `schema.Set`.